usage/local-api.md

```diff
@@ -26,18 +26,17 @@ The Local API is one method of obtaining data from YouTube. For another method o
 
 ## Modules Used in the Local API
 
-| Name  | License | Functionality | Maintained By the FreeTube Team?
-| ------------- | ------------- | ------------- |
-| [youtube-chat](https://github.com/LinaTsukusu/youtube-chat)  | MIT | Live Chat | No |
-| [youtube-suggest](https://github.com/goto-bus-stop/youtube-suggest)  | Apache-2.0 | Search Suggestions | No |
-| [ytdl-core](https://github.com/fent/node-ytdl-core)  | MIT  | Obtain Video Information | No |
-| [ytpl](https://github.com/TimeForANinja/node-ytpl)  | MIT  | Playlists | No |
-| [node-ytsr](https://github.com/TimeForANinja/node-ytsr)  | MIT  | Search Functionality | No |
```

Comment:
Why was ytsr removed?

Reply:
https://github.com/FreeTubeApp/FreeTube/blob/development/package.json#L84

Reply:
i must be blind, will add back!